utils/string/truncateOutputString.ts

```diff
@@ -0,0 +1,5 @@
+export function truncateOutputString(text: string, maxLength = 50) {
+  const truncatedText = text.substring(0, maxLength);
+  const isTruncated = truncatedText.length === text.length;
+  return truncatedText + (isTruncated ? '' : '...');
+}
```

Comment:
a consideration is if `maxLength` is meant to be the full length of the string, you'll need consider the length of your suffix:
```suggestion
export function truncateOutputString(text: string, maxLength = 50) {
  const suffix = '...';
  const truncatedText = text.substring(0, maxLength - suffix.length);
  return  text.length > maxLength ? `${truncatedText}${suffix}` : text;
}
```